### PR TITLE
Consul On Kubernetes - Side by Side Section

### PR DESCRIPTION
**Pull request by pbortnick**
🎟️ [Asana Task](https://app.asana.com/0/1200785913255669/1200992155224310/f)
🔍 [Preview Link](https://consul-git-pbside-by-side-hashicorp.vercel.app/consul-on-kubernetes)
💅 [Design Reference](https://www.figma.com/file/0DT6uKPK214eFZtMG0pQUA/Consul-on-Kubernetes?node-id=102%3A6583)

This PR contains the side by side pink and white section of the consul-on-kubernetes page.

This is a feature PR targeting assembly branch: #11047

### CODE REVIEW

File: website/components/block-list/index.tsx [hunk 1]
```diff
@@ -19,7 +19,7 @@ export default function BlockList({ blocks }: BlockListProps) {
             <img src={image} alt={title} />
           </div>
           <div>
-            <h5 className={s.title}>{title}</h5>
+            <h3 className={s.title}>{title}</h3>
```
Comment (pbortnick) [inline website/components/block-list/index.tsx:22]: Updated to follow accessibility standards

File: website/components/side-by-side/style.module.css [hunk 2]
```diff
@@ -0,0 +1,62 @@
+.sideBySide {
+  display: flex;
+  flex-wrap: wrap;
+
+  @media (--large) {
+    flex-wrap: nowrap;
+  }
+
+  & .sideWrapper {
```
Comment (pbortnick) [inline website/components/side-by-side/style.module.css:9]: I extracted some of the styles from g-grid-container and applied it to work for this section

File: website/pages/consul-on-kubernetes/index.tsx [hunk 3]
```diff
@@ -24,36 +27,66 @@ export default function ConsulOnKubernetesPage() {
...
-            {
-              title: 'Kubernetes Monitoring',
-              description:
-                'Obtaining insights on what’s going on and the health of Kubernetes clusters can be complicated. In addition, security issues and vulnerabilities need to be properly tracked.  ',
-              image: require('./images/blocks/monitoring.svg'),
-            },
-          ]}
+        <SideBySide
+          left={
+            <div>
```
Comment (BrandonRomano) [inline website/pages/consul-on-kubernetes/index.tsx:33]: If this div doesn't do anything other than just allow you to pass in multiple elements here, you could use a fragment.  `<></>`, to accomplish the same thing.

Cool with the extra div though if it serves ya a purpose.

**Review by BrandonRomano — APPROVED** [review]
Looking great ❇️

**Review by BrandonRomano — APPROVED** [review]

**Pull request merged**